packages/vue/examples/search-showcase/movie-search-ai-demo/.prettierrc.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
```

Comment:
@SavvyShah What's the reason to use a different lint and prettier config for this demo?